modules/cudf/src/series.ts

```diff
@@ -481,6 +481,15 @@ export class AbstractSeries<T extends DataType = any> {
    * @summary Hook for specialized Series to override when constructing from a C++ Column.
    */
   protected __construct(inp: Column<T>): Series<T> { return Series.new(inp); }
+
+  value_counts(): DataFrame {
```

Comment:
```suggestion
  value_counts() {
```

Reply:
from an offline discussion with Paul, removing this return type causes some other type issues that will require some updates to resolve. Paul will open a separate PR for this while I continue work on `Series.unique`